templates/flux.yaml

```diff
@@ -108,7 +108,7 @@ spec:
         args:
         - --git-url=$ASB_GIT_REPO
         - --git-branch=$ASB_GIT_BRANCH
-        - --git-path=$ASB_GIT_PATH
```

Comment:
can we use /deploy/bootstrap instead of common?

we've been talking about "bootstrapping" a cluster in terms of getting all the core services deployed and running

Reply:
agreed